app/models/item.ts

```diff
@@ -0,0 +1,22 @@
+export class ItemModel {
+  uuid: string;
+  mods: any;
+  capture: any;
+  typeOfResource: string;
+  title: string;
+  isPDF: boolean;
+  isBook: boolean;
+  isSingleCapture: boolean;
+
+  constructor(data: any) {
+    this.uuid = data.nyplAPI.request.uuid;
+    this.mods = data.nyplAPI.response.mods;
```

Comment:
This answers my question regarding having two API functions, you need both `request` and `response`. Very nitpicky but you can do `const { nyplAPI } = data;` just to not repeat `data.nyplAPI`, or even
`const { nyplAPI: { request, response } } = data;` so you can use `request` and `response` directly.

Reply:
mmm the code above is not working for me?

Reply:
nvm... got it to work